src/plugins/streaming-plugin.js

```diff
@@ -458,15 +464,21 @@ class StreamingHandle extends Handle {
    * @param {string} [params.secret]
    * @returns {Promise<module:streaming-plugin~STREAMING_EVENT_OK>}
    */
-  async stopRecording({ id, audio = true, video = true, data = true, secret }) {
+  async stopRecording({ id, audio = true, video = true, data = true, media, secret }) {
```

Comment:
- jsdocs: use a generic optional array of objects and start property description with `[multistream] Blah Blah`.